sandbox.html

```diff
@@ -474,9 +474,33 @@
     <label for="embeddingTargetId">Embedding target ID of the map view *</label>
     <input name="embeddingTargetId" id="embeddingTargetId" required>
   </div>
+  <div>
+    <label for="geometry">Additional layers (GeoJSON)</label>
+    <textarea name="additionalLayers" id="additionalLayers" rows="10" placeholder="featureCollections array"></textarea>
+    <small>
+      [{
+      "name":"freiburg",
+      "type":"geojson",
+      "content":{"type":"FeatureCollection","features":[
+      {"type":"Feature","properties":{},
+      "geometry":{
+      "coordinates":[[[7.965977481550681,47.93787211649922],[7.903787681150362,47.64741821336713],[8.310793134690243,47.70344757260733],[7.965977481550681,47.93787211649922]]],
+      "type":"Polygon"
+      }
+      }
+      ]}
+      }]
+    </small>
```

Comment:
These examples are hard to read in the sandbox, because there are no line breaks and indentation. Proposal:
```html
    <small>
      <strong>Example:</strong>
      <pre>
[
  {
    "type":"geojson",
    "name":"Freiburg",
    "content": {
      "type": "FeatureCollection",
      "features": [ ... ]
    }
  }
  ...
]</pre>
    </small>
```